.gitignore

```diff
@@ -1,3 +1,4 @@
 .bundle
 *.gem
+.idea
```

Comment:
This stuff should go in `~/.gitignore_global`, not in the repository gitignore.

Reply:
I put it there because for some repos, I do check it in, and it's not possible to selectively ignore via the global gitignore (and jetbrains doesn't allow you to change the location).

But I deleted it, I can manually comment the global entry as needed depending on the repo I work on, no worries.